examples/rollback/rollback.bal

```diff
@@ -44,3 +53,21 @@ function doUpdate(Update u) returns error? {
 
     return;
 }
+
+function incrementSalary(int[] salaryList) returns error? {
+    transaction {
+        foreach int index in 0 ..< salaryList.length() {
+            salaryList[index] += 100;
+        }
+
+        // If the new total salary exceeds `1000`, then the rollback statement performs 
```

Comment:
Do we need `1000` within backticks? :thinking: `doUpdate()` function mentions `-1500` without the backticks.  Maybe we can update both to be consistent with whatever the preferred way.